config/management/src/validator_config.rs

```diff
@@ -52,6 +53,32 @@ pub fn encode_validator_address(
     })
 }
 
+pub fn decode_validator_address(
```

Comment:
doesn't seem used?

Reply:
It's now used.  Basically to rotate a network key, you have to decrypt the validator address, rotate the key, and then re-encrypt it.